spec/fixtures/countries.yml.erb

```diff
@@ -7,8 +7,9 @@
   founded_on: 1867-07-01
   updated_at: 2014-02-24T19:08:06-05:00
   nato: true
-  king: Elisabeth II
+  king: Charles III
```

Comment:
I appreciate the thought, but I think this PR would be better without this unrelated change.